src/lib/updateSiteUrlToLocal.ts

```diff
@@ -15,7 +15,9 @@ export const updateSiteUrlToLocal = async ( siteId: string ) => {
 		return;
 	}
 
-	const studioUrl = `http://localhost:${ server.details.port }`;
+	const studioUrl = server.details.customDomain
```

Comment:
Would it make sense to extract it to its own object method?

Reply:
Maybe… If we could add something like a `getUrl()` method to `server.details`, it'd be a no-brainer. But `server.details` is a typed `SiteDetails` object (exactly the same shape as in the config file), so there's no clear-cut opportunity to start adding methods to that object. 

We could also add a helper function like `getSiteUrl( site: SiteDetails ) => string`, but given how short and intuitive the current logic is and the inherent overhead in adding a new module, I'm not convinced we gain that much from it.

Reply:
Adding a method to the `server` object (a `SiteServer` instance) could work, but there are other places in the code where we need to run an equivalent check and we typically have access to a `SiteDetails` object rather than a `SiteServer` instance. If we can't use the new method/function everywhere, it also doesn't make sense, IMO.

Reply:
OK, I changed my mind 😅 I realized that we'll have HTTPS support soon enough, which will need to be considered here, too. I'll add a helper function

Reply:
Thanks for exploring, those, looks great! 🐥